src/handler.cpp

```diff
@@ -303,7 +303,7 @@ void affect_veh(struct veh_data *veh, byte loc, sbyte mod)
       veh->armor += mod;
       break;
     case VAFF_SEN:
-      veh->sensor = mod;
+      veh->sensor += mod;
```

Comment:
Is += for mod intentional here? This will change vehicle sensor ratings for any vehicle with a sensor mod currently installed.

Reply:
Sensors are entirely NERP atm so can just drop it, currently if you install and uninstall a sensor mod without this + it'll roll back to a sensor value of 252 (Which, again, does nothing, so borked if you do, borked if you don't)

Reply:
Why do vehicles suck so much ;-;